src/Workspaces/Core/Portable/Utilities/TaskExtensions.cs

```diff
@@ -14,6 +14,21 @@ namespace Roslyn.Utilities
     [SuppressMessage("ApiDesign", "CA1068", Justification = "Matching TPL Signatures")]
     internal static partial class TaskExtensions
     {
+        /// <summary>
+        /// Use to explicitly indicate that you are not waiting for a task to complete
+        /// Observes the exceptions from it.
```

Comment:
You'll still get AggregateException on the finalizer instead of a real exception when it finishes.